Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/SmartLiveData.kt

```diff
@@ -38,7 +48,42 @@ class SmartLiveDataProperty<T : Any>(
     }
 }
 
-class SmartLiveData<T : Any>(
+class SmartSingleLiveEvent<T : Any>(
```

Comment:
In that case, the file should be renamed too.
Seems like a strange workaround overall, though.